features/google/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "google-feature",
```

Comment:
Naming nit: `google`? Really? Not google-import, or gle-import?

Reply:
"At the time of writing, we only have the Facebook feature, which is technically called facebookImport, but it does more than just import a users Facebook data - it also analyses and allows the user to explore it."
I think we want to move away from "Import"

Reply:
my-google-data? Can maybe marketing chip in? My main peeve here is the directory name, just "google" is... I don't know. If we can't think about a better alternative, let it be...

Reply:
`features/google` is what it's supposed to be. `features/google-feature` seems a bit weird. Why not `polyPod/polyPod-features/google-polyPod-feature` if we like redundancy so much? :P